plugins/modules/win_route.py

```diff
@@ -37,26 +43,30 @@
   - Works only with Windows 2012 R2 and newer.
 author:
 - Daniele Lazzari (@dlazz)
+- Vicente Danzmann Vivian (@iVcente)
 '''
 
 EXAMPLES = r'''
 ---
-- name: Add a network static route
+- name: Add a network static route.
   community.windows.win_route:
-    destination: 192.168.2.10/32
-    gateway: 192.168.1.1
+    network: 192.168.24.0
+    mask: 255.255.255.0
+    gateway: 192.168.24.1
     metric: 1
     state: present
 
-- name: Remove a network static route
+- name: Remove a network static route.
   community.windows.win_route:
-    destination: 192.168.2.10/32
+    network: 192.168.24.0
+    mask: 255.255.255.0
+    gateway: 192.168.24.1
     state: absent
 '''
 RETURN = r'''
 output:
     description: A message describing the task result.
     returned: always
     type: str
-    sample: "Route added"
+    sample: "Route added!"
```

Comment:
You shouldn't change the output value unless there's a good reason to. You never know what people might be parsing/expecting in their playbooks.